conf/binac.config

```diff
@@ -0,0 +1,25 @@
+/*
+vim: syntax=groovy
+-*- mode: groovy;-*-
+ * ----------------------------------------------------------------------------
+ *  Nextflow config file for use with Singularity on BINAC cluster in Tuebingen
+ * ----------------------------------------------------------------------------
+ * Defines basic usage limits and singularity image id.
+ */
+
+singularity {
+    enabled = true
+}
+
+process {
+    beforeScript = 'module load devel/singularity/2.4.1'
+    executor = 'pbs'
+    container = 'docker://scilifelab/ngi-methylseq'
```

Comment:
Sorry, just noticed this - probably the same on the RNA-seq pipeline. Quite recently I added a new small feature to automatically use the tagged version of the docker container if running a tagged version of the pipeline. Relevant code [here](https://github.com/SciLifeLab/NGI-MethylSeq/blob/master/nextflow.config#L13-L16) and use in a config file [here](https://github.com/SciLifeLab/NGI-MethylSeq/blob/master/conf/uppmax.config#L18).

Reply:
To clarify - just switch this line to this:

```
container = "docker://$wf_container"
```